api/capi/src/nntrainer.cpp

```diff
@@ -24,12 +24,42 @@
 #include "neuralnet.h"
 #include "nntrainer_internal.h"
 #include "nntrainer_log.h"
+#include "parse_util.h"
+#include <stdarg.h>
 #include <string.h>
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+/**
+ * @brief Function to get the number of arguments for specific property
+ * number of arguements = 1
+ *        0 : input_shape
+ *        1 : bias_zero
+ *        2 : normalization
+ *        3 : standardization
+ *        4 : activation
+ *        5 : epsilon
+ * ------------------------------
+ * number of arguements = 2
+ *        6 : Weight Decay ( type & lambda )
+ */
```

Comment:
Store "metadata" (including # arguments of a type) of types in a separated/shared data structure somewhere the type is defined and use it. Having such metadata in a static function is weird.

E.g., after the line where type/enum is defined,
```
const int num_args[ TYPEENUM ] = {
  [ INPUT_SHAPE ] = 1,
 ...
};
```

Reply:
Instead of using this function, it is better to use NULL termination variable for variable arguments. So I remove this function.